File: Model1/Student.cs
namespace Model
{
    public class Student
    {
        public long Id { get; set; }
        public string? FirstName { get; set; }
        public string? LastName{ get; set; }
        public string? IndexNumber { get; set; }
        public List<StudentListensSub>? Subjects { get; set; }
        public MethodOfFinancing methodOfFinancing { get; set; }
        public int YearOfStudy { get; set; }
        public StudyProgram? StudyProgram { get; set; }
        public long StudyProgramId { get; set; }
    }
}

File: ClientApp/src/app/models/student.ts
import { Exam } from "./exam";

export class Student
{
    id:number=0;
    firstname:string="";
    lastname:string="";
    password:string="";
    indexNumber:string="";
    passedExams:Array<Exam>=new Array<Exam>();
    failedExams:Array<Exam>=new Array<Exam>();
    registeredExams:Array<Exam>=new Array<Exam>();
    methodOfFinancing:number=0;
    email:string="";
    yearOfStudy:number=0;
    studyProgramId:number=0;
}